[MSVC]Squirrel.Windows failed to build on MSVC

Hi All,

Squirrel.Windows failed to build on MSVC. Could you please help look at this issue? Thanks in advance.

**Repro steps:**

Open VS2022 x86 Tools command
git clone https://github.com/Squirrel/Squirrel.Windows F:\gitP\Squirrel\Squirrel.Windows
cd F:\gitP\Squirrel\Squirrel.Windows
.\.NuGet\NuGet.exe restore  -ConfigFile F:\tools\Nuget_Config\nuget.config Squirrel.sln
msbuild /m /p:Platform="Any CPU" /p:Configuration=Release Squirrel.sln /t:Rebuild

**Build log:**

[build.log](https://github.com/Squirrel/Squirrel.Windows/files/8906506/build.log)


**Error info:**
error CS0246: The type or namespace name 'InternalsVisibleToAttribute' could not be found (are you missing a using directive or an assembly reference?) 
error CS0400: The type or namespace name 'System' could not be found in the global namespace (are you missing an assembly reference?) [F:\gitP\Squirrel\Squirrel.Windows\src\Squirrel\Squirrel.csproj]
error CS0518: Predefined type 'System.Boolean' is not defined or imported [F:\gitP\Squirrel\Squirrel.Windows\src\Squirrel\Squirrel.csproj]
error CS0246: The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?) 
error CS0246: The type or namespace name 'Microsoft' could not be found (are you missing a using directive or an assembly reference?) [F:\gitP\Squirrel\Squirrel.Windows\src\Squirrel\Squirrel.csproj]
error CS0518: Predefined type 'System.Object' is not defined or imported [F:\gitP\Squirrel\Squirrel.Windows\src\Squirrel\Squirrel.csproj]
error CS0518: Predefined type 'System.String' is not defined or imported [F:\gitP\Squirrel\Squirrel.Windows\src\Squirrel\Squirrel.csproj]

--- a/README.md
+++ b/README.md
@@ -39,8 +39,7 @@ For the impatient:
 ```sh
 git clone --recursive https://github.com/squirrel/squirrel.windows
 cd squirrel.windows
-.\.NuGet\NuGet.exe restore
-msbuild /p:Configuration=Release
+devbuild.cmd
 ```
 See [Contributing](docs/contributing/contributing.md) for additional information on building and contributing to Squirrel.
 

--- a/docs/contributing/building-squirrel.md
+++ b/docs/contributing/building-squirrel.md
@@ -11,11 +11,10 @@ For the Impatient:
 git clone https://github.com/squirrel/squirrel.windows
 cd squirrel.windows
 git submodule update --init --recursive       ## THIS IS THE PART YOU PROBABLY FORGOT
-.\.NuGet\NuGet.exe restore
-msbuild /p:Configuration=Release
+devbuild.cmd
 ```
 
-**Tip:** You can compile the Squirrel.Windows solution with Visual Studio version 2013 and above (including community edition).
+**Tip:** You can compile the Squirrel.Windows solution with Visual Studio version 2019 and above (including community edition).
 
 **Tip:** For Visual Studio versions that use the Visual Studio Installer (2017/2019 and above), you will need to have at least both Desktop .NET development and Desktop C++ development workloads checked in the Visual Studio Installer. You will also need to make sure that the individual package for the VC++ version used by Squirrel is checked.
 

